src/backend/db/products.js

```diff
@@ -8,23 +8,107 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    categories: "men",
+    image: "/Products-images/HRX-Yellow-Printed-Round-Neck-T-Shirt--1.webp",
+    brand: "HRX",
+    name: "Yellow Printed Round Neck",
+    discoutPrice: 700,
+    actualPrice: 900,
+    discout: 40,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 4.5,
   },
   {
     _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    categories: "men",
+    image:
+      "/Products-images/-Men-Green-Slim-Fit-Solid-Casual-Shirt-358156645-1.webp",
+    brand: "Levis",
+    name: "Grey Plain Shirt",
+    discoutPrice: 1200,
+    actualPrice: 2000,
+    discout: 20,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 3.5,
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    categories: "men",
+    image: "/Products-images/Leather-Jacket1.webp",
+    brand: "H&M",
+    name: "Brown Leather Jacket",
+    discoutPrice: 3999,
+    actualPrice: 7999,
+    discout: 50,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 4.5,
+  },
+  {
+    _id: uuid(),
+    categories: "men",
+    image: "/Products-images/Nautica-Men-Tshirts-1.webp",
+    brand: "Nautica",
+    name: "Red Plain Kolor Neck",
+    discoutPrice: 999,
+    actualPrice: 1200,
+    discout: 10,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 4.5,
+  },
+  {
+    _id: uuid(),
+    categories: "Women",
+    image: "/Products-images/-Women-Kurta-Sets2-1.webp",
+    brand: "Nautica",
+    name: "Yellow Printed Long Suit",
+    discoutPrice: 2700,
+    actualPrice: 3000,
+    discout: 20,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 4.5,
+  },
+  {
+    _id: uuid(),
+    categories: "Women",
+    image: "/Products-images/women-kurti-nautrica.webp",
+    brand: "Nautica",
+    name: "Yellow Silk Design Kurta",
+    discoutPrice: 1199,
+    actualPrice: 1500,
+    discout: 30,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 3.5,
+  },
+  {
+    _id: uuid(),
+    categories: "Women",
+    image: "/Products-images/womne-top-hrx.webp",
+    brand: "H&M",
+    name: "Black Cotton Shrug",
+    discoutPrice: 999,
+    actualPrice: 1200,
+    discout: 30,
+    inStock: true,
+    fastDelivery: true,
+    ratings: 3.5,
+  },
+  {
+    _id: uuid(),
+    categories: "Women",
+    image: "/Products-images/long-cote-Women.webp",
+    brand: "Roadster",
+    name: "Dark Blue long Jacket",
+    discoutPrice: 5999,
```

Comment:
Discounted price can be calculated based on actual price and discount.

Reply:
yes , this is just dummy price